test/file_reader-spec.js

```diff
@@ -0,0 +1,63 @@
+'use strict';
+
+// const FileDB = require('../asset/compressed_file_reader/filedb');
+const fixtures = require('jest-fixtures');
+const { newReader, newSlicer } = require('../asset/file_reader/index.js');
+
+const logger = {
```

Comment:
`terascope/utils` has a debug logger which matches the bunyan spec and can be externally enabled via `export DEBUG='teraslice*'` when running tests

For example:

```js
const { debugLogger } = require('@terascope/utils');

// ...
const logger = debugLogger('file_reader-spec');
```

Reply:
replaced logger in test with `debugLogger`